src/steamship/agents/tools/video_generation/did_video_generator_tool.py

```diff
@@ -0,0 +1,120 @@
+"""Tool for generating images."""
+from typing import Any, List, Optional, Union
+
+from steamship import Block, Task
+from steamship.agents.schema import AgentContext
+from steamship.agents.tools import VideoGeneratorTool
+from steamship.utils.repl import ToolREPL
+
+DEFAULT_SOURCE_URL = "https://www.steamship.com/images/agents/man-in-suit-midjourney.png"
+
+
+class DIDVideoGeneratorTool(VideoGeneratorTool):
+    """Tool to generate talking avatars from text using D-ID."""
+
+    name: str = "DIDVideoGeneratorTool"
+    human_description: str = "Generates an a video of you speaking a response to a user."
+    agent_description = (
+        "Used to generate a video of you from text. Use if the user has asked for a video response.  "
+        "The input is the text that you want to say. "
+        "The output is the video of you saying it."
+    )
+    generator_plugin_handle: str = "did-video-generator"
+    generator_plugin_config: dict = {}
+
+    source_url: Optional[str] = DEFAULT_SOURCE_URL
+    """The URL of the source image to be animated."""
+
+    stitch: bool = True
```

Comment:
what is the implication of setting this to `False` ?

Reply:
No clue. It's on the D-ID plugin so I added it here to match. For this and the comments about the other config fields, I'm going to wait until V2 to address; agree with them all but it sounds like @EniasCailliau has a config object that he's working on that we might be able to just use directly for a 1:1 documented, pydantic-friendly obj